This patch adds papi support for AMD Family 10h RevE cpus. There are
no new events compared to RevD.

Signed-off-by: Robert Richter <robert.richter@amd.com>
---
 src/papi_events.csv |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/papi_events.csv b/src/papi_events.csv
index 9ebc16c..e4334f2 100644
--- a/src/papi_events.csv
+++ b/src/papi_events.csv
@@ -103,6 +103,7 @@ CPU,AMD64 (Barcelona RevC)
 CPU,AMD64 (Family 10h RevB Barcelona)
 CPU,AMD64 (Family 10h RevC Shanghai)
 CPU,AMD64 (Family 10h RevD Istanbul)
+CPU,AMD64 (Family 10h RevE)
 #
 PRESET,PAPI_L1_ICH,DERIVED_SUB,INSTRUCTION_CACHE_FETCHES,INSTRUCTION_CACHE_REFILLS_FROM_SYSTEM,INSTRUCTION_CACHE_REFILLS_FROM_L2
 PRESET,PAPI_L1_ICM,NOT_DERIVED,INSTRUCTION_CACHE_MISSES
-- 
1.7.3.4


_______________________________________________
Perfapi-devel mailing list
Perfapi-devel@eecs.utk.edu
http://lists.eecs.utk.edu/mailman/listinfo/perfapi-devel
